src/cpdb/helpers.py

```diff
@@ -1,10 +1,17 @@
+from distutils.command.build import build
 import pandas as pd
 from dotenv import load_dotenv
 import geopandas as gpd
 from src.digital_ocean_client import DigitalOceanClient
 
 load_dotenv()
 
+cpdb_published_version = [
+    "2022-10-25",  # '22 CPDB Adopted
```

Comment:
This would require more code, but would it be possible to show the user those "Adopted", "Executive" etc labels? Each previous build could be represented with a tuple of `(date, alias)`